### DISCUSSION TITLE
Pentest Reports made with SysReptor and ❤️

### DISCUSSION THREAD

**Discussion by aronmolnar** [Syslifters/sysreptor#20] [Show and tell]
We'd love to see what the reports look like that you created with SysReptor.

Here are the Demo reports that are by default imported to your installations:

[Margherita Demo.pdf](https://github.com/Syslifters/sysreptor/files/11459436/Margherita.Demo.pdf)
[Calzone Demo.pdf](https://github.com/Syslifters/sysreptor/files/11459435/Calzone.Demo.pdf)
[OSCP Exam Report Demo.pdf](https://github.com/Syslifters/sysreptor/files/11459434/OSCP.Exam.Report.Demo.pdf)

**Comment by aronmolnar** [comment]
This is a Active Directory demo report from our own company Syslifters made with SysReptor:

[Demo-Report_Syslifters_AD_v1.0_en.pdf](https://github.com/Syslifters/sysreptor/files/11459439/Demo-Report_Syslifters_AD_v1.0_en.pdf)

**Reply by MWedl** [reply]
More demo reports are available here: https://docs.sysreptor.com/demo-reports/

**Comment by kennystrawnmusic** [comment]
## ISP Security LLC ― Internal

* PDF: [ispsec-report-preview.pdf](https://github.com/user-attachments/files/23656940/ispsec-report-preview.pdf)
* Tar archive: [design-ISP-Security_-Internal-Penetration-Test-v0.1.tar.gz](https://github.com/user-attachments/files/23656943/design-ISP-Security_-Internal-Penetration-Test-v0.1.tar.gz)

Will be submitting additional entries for external, web, cloud, WiFi, and others.

**Reply by kennystrawnmusic** [reply]
Oops, forgot to add: [Company Website](https://ispsecurity.co)

**Reply by kennystrawnmusic** [reply]
Submitted pull request #559 with the internal template; will be making more where that came from.

**Comment by didntchooseaname** [comment]
# Altered Security Design Update
Updated the previous version from V1.0 to V2.0

- PDF: https://github.com/didntchooseaname/Altered-Security-Reporting/blob/main/assets/AS-reporting-demo.pdf
- Tar Archive: https://github.com/didntchooseaname/Altered-Security-Reporting/releases/latest/download/design-Altered-Security.tar.gz

**Reply by MWedl** [reply]
Thank you. We will update the design in the docs.
